src/core/game/RailNetwork.ts

```
@@ -0,0 +1,327 @@
+      to,
+      5000,
+      20,
+      false,
+      3,
+    );
+    let tiles: TileRef[] = [];
+    switch (this.railAStar.compute()) {
+      case PathFindResultType.Completed:
+        tiles = this.railAStar.reconstructPath();
+        break;
+      case PathFindResultType.Pending:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)